README.md

```diff
@@ -65,9 +65,6 @@ spec:
 $ sudo ./bin/imageplugin --endpoint tcp://127.0.0.1:10000 --nodeid CSINode -v=5
 ```
 
-### Test using csc
-Get ```csc``` tool from https://github.com/rexray/gocsi/tree/master/csc
-
 ### Mount the image
 $ csc -e tcp://127.0.0.1:10000 node publish abcdefg --vol-context image=kfox1111/misc:test --cap MULTI_NODE_MULTI_WRITER,block --target-path /tmp/csi
```

Comment:
If csc tool is going away, then the subsequent lines describing csc should probably go too?